packages/ui-react/src/components/ChooseOfferForm/ChooseOfferForm.tsx

```diff
@@ -54,7 +54,7 @@ const OfferBox: React.FC<OfferBoxProps> = ({ offer, selected, onChange }: OfferB
   };
 
   const renderOption = ({ title, periodString, secondBenefit }: { title: string; periodString?: string; secondBenefit?: string }) => (
-    <div className={styles.offer} role="option">
+    <div className={styles.offer}>
```

Comment:
I'm hesitant about removing the `role="option"`, let's only do that if it makes the options more accessible on screen readers! I can imagine without it, the screen reader can have difficulties with it because the `<label>` has a lot f HTML in it (which is invalid HTML, which apparently axe devtools does not see)

Reply:
Yes!

Reply:
I did a experiment on IOS and android. My results:

Android: In both situations (with or without role="option") the accessibility is bad. 
iOS: With the role="option" is the block gets read as a whole and on a swipe I navigate directtly to the other offer block (this is what we want).

Therefor I suggest keeping it.

I created a ticket to optimize this futhert in the future: [OTT-772](https://videodock.atlassian.net/browse/OTT-772)